docs/wxm-devices/wifi-m5-bundle/m5-install-weather-station.mdx

```diff
@@ -51,35 +51,36 @@ Don't sure for your installation point? Check deployment examples [here](/deploy
 <div class="explanation">
   <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/max-pole-diameter.svg")} />
     <p>
-      The pole’s diameter should be at least 25mm/1in and maximum 50mm/2in for optimal use. Suggested material should be aluminium.
-    </p>
+      The pole’s height, on which the weather station will be mounted, should be at least 2m/6.5ft.
+      This means that the weather station should be placed at least 2m/6.5ft above the ground and any other horizontal
+      or almost horizontal surface (e.g., roof) above the station. The maximum acceptable pole height is 5m/16.5ft.    </p>
 </div>
 
 <hr/>
 
 ### Tighten Pole
-
+<p>
+  The pole should be adequately tightened in order to prevent any pole oscillations in cases of strong winds.
+      This will prevent the rain gauge spoon from swinging and collecting faulty precipitation measurements.
+ </p>     
 <div class="control">
   <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/ws1000-tighten-pole.svg")} />
 </div>
 
-<p>
-The pole should be adequately tightened in order to prevent any pole oscillations in cases of strong winds.
-This will prevent the rain gauge spoon from swinging and collecting faulty precipitation measurements.
-</p>
+
 
 <hr/>
 
 ### Installation Point
 
 The pole together with the outdoor sensor should be placed in an installation point, which:
```

Comment:
Replace it with:
`The pole together with the weather station should be placed in an installation point, which:`

Please also use the same text on WS2000.